docs/getting_started/dashboard/dashboard_block_reference.md

```diff
@@ -92,13 +92,8 @@ The table contains the following columns: Name, Image, Code, Category, Type, Var
 On the **Properties** tab, set values in the following fields:
 
 - **Name** - Enter a name for the block.
-<<<<<<< HEAD
 - **Number of products to be displayed** - Set a number of products to be displayed. Default value = 10.
 - **Stock** - Set up the maximum stock value (only products with stock number greater than zero and less than the set maximum number will be displayed). Default value = 10.
-=======
-- **Number of visible products** - Set a number of products to be displayed. Default value = 10, minimum value = 1, and maximum = 10.
-- **Stock threshold** - Set up the maximum stock value (only products with stock number greater than zero and less than the set maximum number are displayed). Default value = 10.
```

Comment:
I would have kept those ones.

Reply:
![products-with-lowest-stock](https://github.com/user-attachments/assets/d519b50d-159f-4c6d-b681-559b659da246)

Reply:
Added in #246, it was updated in #273

https://github.com/ibexa/documentation-user/pull/273/files#diff-47c91238da8a14c8019ac6e522f1b39f57175b8bef938b372f556d7d3c01e671L96-R97

Reply:
This is targeted at 4.6 branch, not master

Reply:
> This is targeted at 4.6 branch, not master

@dabrt 
My screenshot is from 4.6.16.
It was written when master was still equal to 4.6.
Nothing to do with 5.0 here.